test/cases/api/common/common.sh

```diff
@@ -19,6 +19,20 @@ function _instanceCheck() {
   echo "✔️ Instance checking"
   local _ssh="$1"
 
+  # Retry loop to wait for instance to be ready
```

Comment:
I'd be tempted to add an initial delay shorter than 30s so that there is some chance of catching it on the first try and not always waiting 30s -- but I also don't know how long it takes these things to startup :)

Reply:
Well it will try it once right away before waiting which should work in the vast majority of cases. I'm aware of only the gcp test on 8.10-ga where it takes so long that it fails. The alternative there was to wait for the whole test run again which is ~30 minutes, so 30 seconds compared to that is nothing and I'd prefer to not complicate this :)